resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -560,4 +580,6 @@ private[spark] object Config extends Logging {
   val KUBERNETES_VOLUMES_OPTIONS_SERVER_KEY = "options.server"
 
   val KUBERNETES_DRIVER_ENV_PREFIX = "spark.kubernetes.driverEnv."
+
+  val KUBERNETES_LABEL_MAX_LENGTH = 63
```

Comment:
This looks misleading. Are you trying to use this for K8s Label only?

Reply:
Picked the more general one between [LabelValueMaxLength](https://github.com/kubernetes/apimachinery/blob/master/pkg/util/validation/validation.go#L159)  and [DNS1123LabelMaxLength](https://github.com/kubernetes/apimachinery/blob/master/pkg/util/validation/validation.go#L181), or we follow https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names to name it `KUBERNETES_DNSNAME_MAX_LENGTH`

Reply:
I'd say `KUBERNETES_DNSNAME_MAX_LENGTH` sounds good :)